cmake/SeracMacros.cmake

```diff
@@ -125,3 +125,41 @@ macro(serac_convert_to_native_escaped_file_path path output)
     file(TO_NATIVE_PATH ${path} ${output})
     string(REPLACE "\\" "\\\\"  ${output} "${${output}}")
 endmacro(serac_convert_to_native_escaped_file_path)
+
+##------------------------------------------------------------------------------
+## serac_add_tests( SOURCES       [source1 [source2 ...]]
+##                  DEPENDS_ON    [dep1 [dep2 ...]]
+##                  NUM_MPI_TASKS [num tasks])
+##
+## Adds tests from the given sources
```

Comment:
clarification: Creates an executable per given source and then adds the test to CTest